src/Engine/Renderer/Renderer.cpp

```diff
@@ -431,6 +429,7 @@ namespace Ra
             m_pickingFbo->attachTexture( GL_COLOR_ATTACHMENT0, m_pickingTexture.get()->texture() );
             if ( m_pickingFbo->checkStatus() != GL_FRAMEBUFFER_COMPLETE )
             {
+                LOG( logERROR ) << "FBO info : " << m_pickingFbo->checkStatus();
                 LOG( logERROR ) << "FBO Error : " << m_pickingFbo->checkStatus();
             }
```

Comment:
doublon?

Reply:
Well, I don't remember this modification ... Nevertheless I'll remove this copy copy/paste side effect on the next commit ... For the moment, this PR is quite perfectible on Linux ... I'm working on that ...